### ISSUE TITLE
Create a results subfolder

### ISSUE THREAD

**Issue by jasonrobwebster** [Percept-Health-Solve/seir-model#20]
Some results should be provided with the model. At the moment, the `data/` folder is used for storing calibration data, as well as all output for our results.

**Comment by hellwind495** [comment]
Jason,

The lack of checking whether an output folder exists also creates a problem when specifying the "--output_dir" on the CLI.

**Comment by jasonrobwebster** [comment]
Yup, agreed. Trying to address this in the refactor by checking if the folder exists, and if not, creating one.

**Comment by hellwind495** [comment]
Thanks, didn't seem to run when I tried the starting script. I'll bug test it in detail.

Maybe we'll discuss it in the team meeting, but is there a plan to give more output on a custom run? For example, give a progress update such as "data import done, now starting fitting"?
